### ISSUE TITLE
add efficientnetv2-s

### ISSUE THREAD

**Issue by sukyoungjeong-furiosa** [furiosa-ai/furiosa-models#106]
This is a tracking issue for adding model: `efficientnet_v2_s`

## Checklist
- [x]  Identify the model structure.
- [x]  Perform quantization and calibration of the model.
  - Accuracy: `75.208%` (original f32 model: `84.228%`)
  - https://github.com/furiosa-ai/furiosa-models/pull/130
- [x]  Deploy an initial model implementation, along with compiled IRs and sanity-checking tests.
- [x]  Implement pre/post-processing codes and regression tests.
- [x]  Add unit tests, example code, preliminary documentation.
- [ ]  Refine and improve the overall code structure.
- [x]  Enhance and proof-read documentation.

## About the model
- model source: [pytorch efficientnet_v2_s](https://pytorch.org/vision/main/models/generated/torchvision.models.efficientnet_v2_s.html)
- ```python
  import torch
  import onnx
  from tempfile import NamedTemporaryFile
  import furiosa.optimizer.frontend.onnx

  torch_model = torch.hub.load(
      "pytorch/vision:v0.14.0", "efficientnet_v2_s", weights="DEFAULT", skip_validation=True, force_reload=True
  )
  torch_model = torch_model.eval()
  with NamedTemporaryFile(suffix=".onnx") as file:
      torch.onnx.export(
          torch_model, torch.randn(1, 3, 384, 384), file, opset_version=13
      )
      onnx_model = onnx.load_model(file.name)

  onnx.shape_inference.infer_shapes(onnx_model, check_type=True, strict_mode=True)
  onnx.checker.check_model(onnx_model)

  optimized_onnx_model = furiosa.optimizer.frontend.onnx.optimize_model(onnx_model)
  onnx.checker.check_model(optimized_onnx_model, full_check=True)
  optimized_onnx_model = optimized_onnx_model.SerializeToString()
  ```
- input: 3-channel (BGR) image of 384x384 `numpy.float32` (Memory format: NCHW)
- output: `numpy.float32` tensor with the shape ([1,1000])
- accuracy: `75.208%` (original f32 model: `84.228%`)
